docs/sdkx_android/release-notes.mdx

```diff
@@ -23,6 +23,23 @@ import {
 
 <LatestSdkVersion>
 
+## 10.3.0
+
+_October 30th, 2023_
+
+** Release Notes **
+
+- Added support for Android 14
+- Moved to AndroidX support libraries. SDK X will not compile with legacy Android Support libraries anymore.
+- Added support for multiple attachments in conversations (Support for multiple attachment in bot upload step will be added in Q1`24)
+- Added a new API to fetch SDKX version. For more details, [refer here](/sdkx_android/support-tools#get-sdk-version).
+- Added a new API to close active SDK session. For more details, [refer here](/sdkx_android/support-tools#close-session).
+- Added a new delegate `agentMessageReceived` when agent sends a message. For more details, [refer here](/sdkx_android/delegates#agent-message-received).
+- `Helpshift.login()` API now return a boolean indicating whether login was successful or not. This change is in existing API to introduce mandatory validations for login parameters and hence this might affect existing code compilation. For more details, [refer here](/sdkx_android/users#logging-in-users).
+- Added support for attachments in Smart FAQs.
+- Add Minimum supported OS by SDK X to Android 7 ie API Level 24
```

Comment:
It should be 
Minimum supported OS by SDK X is moved to Android 7 i.e, API Level 24

Reply:
agree